EasyCommands/BlockHandlers/TurretBlockHandlers.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Sandbox.Game.EntityComponents;
+using Sandbox.ModAPI.Ingame;
+using Sandbox.ModAPI.Interfaces;
+using SpaceEngineers.Game.ModAPI.Ingame;
+using System.Collections.Generic;
+using System.Collections;
+using System.Linq;
+using System.Text;
+using System;
+using VRage.Collections;
+using VRage.Game.Components;
+using VRage.Game.GUI.TextPanel;
+using VRage.Game.ModAPI.Ingame.Utilities;
+using VRage.Game.ModAPI.Ingame;
+using VRage.Game.ObjectBuilders.Definitions;
+using VRage.Game;
+using VRage;
+using VRageMath;
+
+namespace IngameScript {
+    partial class Program {
+        public class TurretControlBlockHandler : FunctionalBlockHandler<IMyTurretControlBlock> {
+            public TurretControlBlockHandler() {
+                AddNumericHandler(Property.RANGE, b => b.Range, (b, v) => b.Range = v, 100f);
+                AddPropertyHandler(Property.LOCKED, TerminalPropertyHandler("EnableTargetLocking", true));
+                AddPropertyHandler(Property.AUTO, TerminalPropertyHandler("AI", true));
+                AddBooleanHandler(Property.USE, b => b.IsUnderControl);
+                AddReturnHandlers(Property.TARGET, Return.VECTOR,
+                    TypeHandler(VectorHandler(b => {
+                        Vector3D? target = Vector3D.Zero;
+                        if (b.HasTarget) target = GetPosition(b.GetTargetedEntity());
+                        return target.Value;
```

Comment:
I am a bit confused why that is not simply 
```
b => b.HasTarget ? GetPosition(b.GetTargetEntity()) : Vector3D.Zero;
```

Reply:
I copied from other and didn't consider that it can be simplified, good callout.